evaluate/evaluate_test.go

```diff
@@ -27,6 +27,7 @@ import (
 	modeltesting "github.com/symflower/eval-dev-quality/model/testing"
 	"github.com/symflower/eval-dev-quality/provider"
 	providertesting "github.com/symflower/eval-dev-quality/provider/testing"
+	"github.com/symflower/eval-dev-quality/task"
```

Comment:
Please update the README as well with these changes. We now have a generic task interface.